config/locales/zh-CN.yml

```diff
@@ -605,12 +619,13 @@ zh-CN:
       mfa:
         multifactor_auth: 多因素验证（MFA）
         otp: 身份认证应用
-        disabled_html: 您尚未启用基于 OTP 的多因素身份验证。请参考 <a target="_blank" rel="noopener" href="https://guides.rubygems.org/setting-up-multifactor-authentication">RubyGems
```

Comment:
Are the changes in this file intentional?

Reply:
bin/fill-locales did that.  Looks like whitespace adjustments.